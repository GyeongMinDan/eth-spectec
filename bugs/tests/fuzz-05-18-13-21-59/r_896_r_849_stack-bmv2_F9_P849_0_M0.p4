header hdr {
   }
    control compute(inout hdr h)() {
     apply {
      list<_> tmp;
    }
   }
