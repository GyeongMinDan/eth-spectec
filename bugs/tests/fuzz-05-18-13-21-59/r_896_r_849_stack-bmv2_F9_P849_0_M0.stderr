likely_hits/r_896_r_849_stack-bmv2_F9_P849_0_M0.p4(3): [--Wwarn=unused] warning: Control compute is not used; removing
    control compute(inout hdr h)() {
            ^^^^^^^
[--Wwarn=missing] warning: Program does not contain a `main' module
